app/assets/javascripts/application.js.erb

```diff
@@ -77,6 +77,34 @@ jQuery(document).ready(function ($) {
     return false;
   });
 
+  $.extend($.datepicker, {
```

Comment:
ARGHH!! @hirtie-maxim please don't add anything new to `application.js.erb`. Adding new scripts here should be considered deprecated. `frontend/app/misc/datepicker-defaults.js` is probably the right file to append to.